tasks/balances.ts

```diff
@@ -1,9 +1,21 @@
-import { getAddress } from "@zetachain/addresses";
-import ZetaEth from "@zetachain/interfaces/abi/json/contracts/Zeta.eth.sol/ZetaEth.json";
+import { networks } from "@zetachain/networks";
+import { getAddress } from "@zetachain/protocol-contracts";
+import ZetaEth from "@zetachain/protocol-contracts/abi/evm/Zeta.eth.sol/ZetaEth.json";
+import ZRC20 from "@zetachain/protocol-contracts/abi/zevm/ZRC20.sol/ZRC20.json";
+import * as bitcoin from "bitcoinjs-lib";
 import * as dotenv from "dotenv";
-import { ethers } from "ethers";
+import ECPairFactory from "ecpair";
 import { task } from "hardhat/config";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
+import ora from "ora";
+import * as ecc from "tiny-secp256k1";
+
+declare const hre: any;
+
+dotenv.config();
+
+const BTC_API = "https://blockstream.info/testnet/api";
```

Comment:
should this bitcoin network/api information be in a config / package?

Reply:
Done: https://github.com/zeta-chain/toolkit/pull/10/commits/472f54f1f02499d7e729af4bc0e8301a36b3b02c

I can't publish an updated version of `@zetachain/publish` for some reason, though:

https://github.com/zeta-chain/networks/actions/runs/5518574540/jobs/10062714440

So for now this works only when linking the package directly.

Reply:
https://github.com/zeta-chain/toolkit/pull/10/commits/3d1da4c5bdbd5e915f50ceafa86e4a40d2961dad